tests/keras_autolog/test_keras_autolog.py

```diff
@@ -39,12 +44,19 @@ def random_one_hot_labels(shape):
                       loss='categorical_crossentropy',
                       metrics=['accuracy'])
 
-        model.fit(data, labels, epochs=10)
+        if fit_variant == 'fit_generator':
+            def generator():
+                while True:
+                    yield data, labels
+            model.fit_generator(generator(), epochs=10, steps_per_epoch=1)
+        else:
+            model.fit(data, labels, epochs=10)
 
     return client.get_run(run.info.run_id)
 
 
 @pytest.mark.large
+@pytest.mark.parametrize('fit_variant', ['fit', 'fit_generator'])
```

Comment:
Nice, could we also add this parametrization to `test_keras_autolog_model_can_load_from_artifact` so that we have test coverage verifying that autologging logs a model when `fit_generator` is used?